packages/webview_flutter/webview_flutter_android/test/android_webview_controller_test.mocks.dart

```diff
@@ -870,19 +870,6 @@ class MockAndroidWebViewController extends _i1.Mock
         returnValue: _i9.Future<void>.value(),
         returnValueForMissingStub: _i9.Future<void>.value(),
       ) as _i9.Future<void>);
-
-  @override
-  _i9.Future<void> setOnJavaScriptTextInputDialog(
```

Comment:
https://github.com/flutter/packages/pull/5796 has not landed yet, so I removed these generated overrides to have the `pathified` test pass.